services/user-management/src/main/java/com/foodie/user/controllers/RolePermissionController.java

```diff
@@ -19,16 +18,19 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
 
 @Slf4j
 @RestController
 @CrossOrigin
 @RequestMapping("/api/v1/security/{role}/permissions")
 public class RolePermissionController {
 
-    private final RoleService roleService;
-    private final PermissionService permissionService;
+    public final RoleService roleService;
```

Comment:
Why are you making these `public`? They should be `prive`

Reply:
That was an oversight